website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
 weight: 30
 ---
 
-Making an Operator SDK release involves:
-
-- Updating `CHANGELOG.md` and migration guide.
-- Tagging and signing a git commit and pushing the tag to GitHub.
-- Building a release binary and signing the binary
-- Creating a release by uploading binary, signature, and `CHANGELOG.md` updates for the release to GitHub.
-- Creating a release branch of the form `v1.2.x` for each major and minor release.
-
-Release steps can be found [below](#release-steps). If you have not run a release before we recommend reading
-through the sections directly following this one.
+These steps describe how to conduct a release of the operator-sdk repo using example versions.
+Replace these versions with the current and new version you are releasing, respectively.
 
-## Dependency and platform support
+## Prerequisites
 
-### Go version
+- [`git`][git]
+- [`gpg2`][gpg2] and a [GPG key][gpg-key-create]. If you have both `gpg` and `gpg2` available, make sure the latter is used:
+    ```sh
+    git config --global gpg.program gpg2
+    ```
+- Your GPG key is publicly available in a [public key server][gpg-upload], like https://keyserver.ubuntu.com/.
 
-Release binaries will be built with the Go compiler version specified in the Operator SDK's [prerequisites section][doc-readme-prereqs].
+##### MacOS users
 
-### Kubernetes versions
+Install GNU `sed`, `make`, and `gpg2` which may not be by default:
 
-As the Operator SDK interacts directly with the Kubernetes API, certain API features are assumed to exist in the target cluster. The currently supported Kubernetes version will always be listed in the SDK [prerequisites section][doc-readme-prereqs].
+```sh
+brew install gnu-sed make gnupg
+```
 
-### Operating systems and architectures
+Configure `gpg2` to sign with `git`:
 
-Release binaries will be built for the `x86_64` architecture for MacOS Darwin platform and for the following GNU Linux architectures: `x86_64`, `ppc64le`, `arm64`, and `s390x`.
+```bash
+echo "export GPG_TTY=tty" >> ~/.bashrc
+source ~/.bashrc
+```
 
-Base images for ansible-operator, helm-operator, and scorecard-test will be built for the following GNU Linux architectures: `x86_64`, `ppc64le`, and `arm64`.
+## Major and Minor releases
 
-Support for the Windows platform is not on the roadmap at this time.
+We will use the `v1.3.0` release version in this example.
 
-## Binaries and signatures
+### Before starting
 
-Binaries will be signed using a maintainers' verified GitHub PGP key. Both binary and signature will be uploaded to the release. Ensure you import maintainer keys to verify release binaries.
+A release branch must be created and [mapped][netlify-deploy] _before the release begins_
```

Comment:
Not clear for me what I should be looking for as ref in [mapped][netlify-deploy].